### DISCUSSION TITLE
【324800】Shadow Warrior 2【本体】

### DISCUSSION THREAD

**Discussion by mcluoxw** [pjy612/SteamManifestCache#166] [Share]
appid: 324800
 Key:
```
324801;c7d42c521529297c5a3a3174a89a1fb9084a0ec10f56c3de7c0a5c5c9d9134f1
```
[Shadow Warrior 2.zip](https://github.com/pjy612/SteamManifestCache/files/13766985/Shadow.Warrior.2.zip)

**Comment by mcluoxw** [comment]
> `appid: 324800
Key: 
324801;c7d42c521529297c5a3a3174a89a1fb9084a0ec10f56c3de7c0a5c5c9d9134f1` [Shadow Warrior 2.zip](https://github.com/pjy612/SteamManifestCache/files/13766985/Shadow.Warrior.2.zip)

**Reply by mcluoxw** [reply]
t.me/mcluoxw

**Reply by pjy612** [reply]
code 呢？

**Reply by mcluoxw** [reply]
12A017C85209E9AB不好意思忘记了

**Comment by mcluoxw** [comment]
> code 呢？
12A017C85209E9AB
